models/grading/grading_events.sql

```diff
@@ -18,10 +18,12 @@ select
     splitByString('/', course_id)[-1] as course_key,
     org,
     verb_id,
-    JSONExtractFloat(event, 'result', 'score', 'scaled') as scaled_score
+    coalesce(JSONExtractFloat(event, 'result', 'score', 'scaled'), 0) as scaled_score
 from {{ ref("xapi_events_all_parsed") }}
 where
     verb_id in (
         'http://id.tincanapi.com/verb/earned',
-        'https://w3id.org/xapi/acrossx/verbs/evaluated'
+        'https://w3id.org/xapi/acrossx/verbs/evaluated',
```

Comment:
Should we be checking anything else for `evalauted` events, given that we're including them in the `problem_events` MV as well? Or does it make sense to use those same events in both MVs?

Reply:
I think you are right, those evaluated, passed and failed events are specific for the courses objects. We should be checking for that too.